web/src/classic/components/organisms/Published/core/hooks.ts

```diff
@@ -54,6 +58,22 @@ export default (alias?: string) => {
 
   const tags = data?.tags; // Currently no need to convert tags
 
+  const selectedLayerId = useMemo(() => {
```

Comment:
You are calling this LayerId, but its layerId and featureId. Is it better to rename useSelected's value (currently `selected`) to selectedItems or something, and then call this selected?
Or....maybe something else if you can think of something that makes more sense.